tests/integration.rs

```diff
@@ -4356,6 +4372,68 @@ mod tests {
             handle_child_output(r, &output);
         }
 
+        #[test]
+        #[cfg(target_arch = "x86_64")]
+        fn test_virtio_balloon() {
+            let focal = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
+            let guest = Guest::new(Box::new(focal));
+
+            let kernel_path = direct_kernel_boot_path();
+
+            let api_socket = temp_api_path(&guest.tmp_dir);
+
+            let guest_memory_size_gb = 4;
+
+            //Let's start a 4G guest with balloon occupied 2G memory
+            let mut child = GuestCommand::new(&guest)
+                .args(&["--api-socket", &api_socket])
+                .args(&["--cpus", "boot=4"])
+                .args(&[
+                    "--memory",
+                    format!("size={}G", guest_memory_size_gb).as_str(),
```

Comment:
Simplify with the following:
```suggestion
                .args(&["--memory", "size=4G"])
```